go/oasis-node/cmd/genesis/migrate.go

```diff
@@ -357,6 +358,9 @@ NodeLoop:
 	// Reduce maximum consensus layer block size.
 	newDoc.Consensus.Parameters.MaxBlockSize = 1_048_576 // 1 MiB
 
+	// Add new staking thresholds.
+	newDoc.Staking.Parameters.Thresholds[staking.KindKeyManagerChurp] = *quantity.NewFromUint64(10_000_000_000_000)
```

Comment:
We should make sure that this works with existing genesis files that don't have this threshold set. I believe there is some validation that would currently fail.

Reply:
If I don't set this, the `SanityCheck` of staking parameters will fail as the new threshold kind is not defined. But I don't want to remove it from the valid threshold list.